shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -584,4 +597,16 @@ - (void)scrollWheel:(NSEvent*)event {
   [self dispatchMouseEvent:event phase:kHover];
 }
 
+- (void)onSettingsChanged:(NSNotification*)notification {
+  NSString* brightness =
+      [[NSUserDefaults standardUserDefaults] stringForKey:@"AppleInterfaceStyle"];
+  [_settingsChannel sendMessage:@{
+    @"platformBrightness" : [brightness isEqualToString:@"Dark"] ? @"dark" : @"light",
+    // The values below are hard-coded, but the iOS implementation has heuristics
```

Comment:
Please file a bug for this and make the comment a TODO referencing the bug.

Reply:
Done